sample-app/src/components/FilterSearch.tsx

```diff
@@ -0,0 +1,125 @@
+import { SearchParameterField } from "@yext/answers-headless";
+import { useState } from "react";
+import { useAnswersActions } from '@yext/answers-headless-react';
+import InputDropdown from "./InputDropdown";
+import renderWithHighlighting from "./utils/renderWithHighlighting";
+import { Option } from "./DropdownSection";
+import DropdownSection from "./DropdownSection";
+import { processTranslation } from "./utils/processTranslation";
+import { useSynchronizedSearch } from "../hooks/useSynchronizedSearch";
+
+const SCREENREADER_INSTRUCTIONS = 'When autocomplete results are available, use up and down arrows to review and enter to select.'
+
+export interface FilterSearchProps {
+  title: string,
+  sectioned: boolean,
+  searchFields: Omit<SearchParameterField, 'fetchEntities'>[],
+  screenReaderInstructionsId: string
+}
+
+export default function FilterSearch (props: FilterSearchProps): JSX.Element {
+  const { title, sectioned, searchFields, screenReaderInstructionsId } = props;
+  const answersActions = useAnswersActions();
+  const [input, setInput] = useState('');
+  const searchParamFields = searchFields.map((searchField) => {
+    return { ...searchField, fetchEntities: false }
+  });
+
+  const [filterSearchResponse, executeFilterSearch] = useSynchronizedSearch((inputValue) => {
+    return inputValue
```

Comment:
Rather than using the ternary operator here, can we do something like: `return answersActions.executeFilterSearch(inputValue ?? '', sectioned, searchParamFields);`

Reply:
Updated!